packaging/examples/kafka-access-with-user.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kafka.strimzi.io/v1beta2
```

Comment:
Maybe for a wider discussion. But should we start with `v1beta2`? I do not think we want to really couple this with the other resources. So I wonder if we should start with `v1alpha1` and eventually use different group as well.

Reply:
So I originally had it as just `v1`. But then I was looking at the User reference and in the original proposal we said that if you didn't specify an apiVersion we would assume the apiVersion matched the KafkaAccess version, so that's why I changed it to `v1beta2`. However I do think you have a point that perhaps we shouldn't couple it. At the moment the `apiVersion` and the `kind` given for the user in the KafkaAccess CR is actually ignored and we just assume the kind is KafkaUser so I could change the KafkaAccess apiVersion to `v1alpha1` without any functional impact.

Reply:
After discussion on slack we are going to start with `v1alpha1` and switch the `apiGroup` to `access.strimzi.io`. Then in the User reference we will require a `kind` and `apiGroup`.